maestro-agent/src/main/groovy/org/maestro/agent/ext/requests/abnormaldisconnect/AbnormalDisconnect.groovy

```diff
@@ -0,0 +1,28 @@
+package org.maestro.agent.ext.requests.abnormaldisconnect
+
+import org.maestro.client.exchange.MaestroTopics
+import org.maestro.client.notes.OkResponse
+import org.maestro.common.agent.AgentEndpoint
+import org.maestro.common.client.MaestroClient
+import org.maestro.common.client.notes.MaestroNote
+
+class FlushHandler implements AgentEndpoint{
```

Comment:
@Frawless just as general note, please make sure to name the classes according to the external endpoint they will be handling. For example, AbnormalDisconnectHandler instead of FlushHandler.

Reply:
Good point, I will change it after finishing some school stuffs.